include/cantera/zeroD/Reactor.h

```diff
@@ -108,9 +108,17 @@ class Reactor : public ReactorBase
      * @param[in] y solution vector, length neq()
      * @param[out] ydot rate of change of solution vector, length neq()
      * @param[in] params sensitivity parameter vector, length ReactorNet::nparams()
+     * @deprecated Replaced by eval(double t, double* ydot). To be removed after
+     *     Cantera 2.6.
      */
-    virtual void evalEqs(doublereal t, doublereal* y,
-                         doublereal* ydot, doublereal* params);
+    virtual void evalEqs(double t, double* y, double* ydot, double* params) {
+        throw NotImplementedError("Reactor::evalEqs", "Deprecated");
```

Comment:
I'm relying on `evalEqs` throwing to implement the transitional behavior in `ReactorNet::eval`. What happens there is that if `evalEqs` is implemented, it uses that but supplies a deprecation warning. If `evalEqs` throws, then it knows to use `eval` instead. The logic is that if a user has a `Reactor`-derived class with an implementation of `evalEqs`, we need to use that if possible, otherwise they would silently get the surprising behavior of the base class's `eval` method being called rather than their `evalEqs` method.

Reply:
In this case, a more descriptive error message may be good.